notebooks/212-pyannote-speaker-diarization/212-pyannote-speaker-diarization.ipynb

```diff
@@ -228,7 +242,8 @@
     ">To use this model, you must be a registered user in 🤗 Hugging Face Hub. You will need to use an access token for the code below to run. For more information on access tokens, please refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).\n",
     ">You can log in on HuggingFace Hub in the notebook environment using the following code:\n",
     "```python\n",
-    "## login to huggingfacehub to get access to pre-trained model \n",
+    "<a id=\"3\"></a>\n",
+    "## login to huggingfacehub to get access to pre-trained model [&#8657;](#0)\n",
```

Comment:
```suggestion
    "## login to HuggingFace hub to get access to pre-trained model [&#8657;](#0)\n",
```

Reply:
this too comment in code snippet, not header

Reply:
yes, sorry, fixed